front-ps/README.md

```diff
@@ -36,14 +36,9 @@ yarn
 yarn build:server
```

Comment:
Are `npm` commands similar to these ones with `yarn`?

Reply:
@vrom911 With 5d5fd84 I've updated `README` for using `npm` commands.